javascript-functions/main.js

```diff
@@ -0,0 +1,32 @@
+function convertMinutesToSeconds(minutes) {
+  minutes *= 60;
+  return minutes;
+}
+
+function greet(name) {
+  return 'Hey, ' + name;
+}
+
+function getArea(width, height) {
+  return height * width;
+}
+
+function getFirstName(person) {
+  return Object.values(person)[0];
+}
```

Comment:
Works, but I think just doing `person.firstName` is much easier to read and a lot cleaner of a solution. With your solution, I would have no idea what value you're actually trying to get, without having to do a lot more work to figure it out.

Reply:
Sounds Good, I have now changed it to person.firstName instead of Object.values(person)[0];. Thank you!